buyer-api/src/utils/routes.js

```diff
@@ -28,4 +28,32 @@ const validateAddress = paramName => async (req, res, next) => {
   return undefined;
 };
 
-export { errorHandler, asyncError, validateAddress };
+/**
+ * Middleware to check if an notarization result is valid or not.
+ * @public
+ */
+const validateFields = () => (req, res, next) => {
```

Comment:
There is no need to have a factory function here. You could just export `export const validateFields = (req, res, next) => {...}` and use it in the router